packages/app-mobile/utils/fs-driver-rn.ts

```diff
@@ -351,7 +351,7 @@ export default class FsDriverRN extends FsDriverBase {
 			} else {
 				// the result is an array
 				if (multiple) {
-					result = await DocumentPicker.pickMultiple();
+					result = await DocumentPicker.pick({ allowMultiSelection: true });
```

Comment:
I just follow the example of implementation from the changelog from the documentation [see this PR](https://github.com/rnmods/react-native-document-picker/pull/651/files).
I tested it and it worked fine to attach multiple and just one document.